playbooks/e2e.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- name: E2E test
+  hosts: all
+  vars:
+    cloud_provider: "{{ cloud }}"  # inherited from the parent job
+    project_dir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+    project_tf_dir: "{{ project_dir }}/terraform"
+    prefix: "pr{{ zuul.change }}"
+    testcluster_name: "pr{{ zuul.change }}"
+  environment:
+    ENVIRONMENT: "{{ cloud_provider }}"
+    PATH: "{{ ansible_user_dir }}/.local/bin:{{ ansible_env.PATH }}"
+  tasks:
+  - name: get ssh version
+    command: "ssh -V"
```

Comment:
Why?

Reply:
it is here just to debug the TF file provisioner issue that we are facing right now